agent/exec/dockerapi/docker_client_stub.go

```diff
@@ -85,6 +87,12 @@ func (sa *StubAPIClient) ContainerStop(ctx context.Context, containerID string,
 	return sa.ContainerStopFn(ctx, containerID, timeout)
 }
 
+// NodeInspectWithRaw is part of the APIClient interface
+func (sa *StubAPIClient) NodeInspectWithRaw(ctx context.Context, nodeID string) (swarm.Node, []byte, error) {
```

Comment:
It was a leftover from my attemps at using NodeInspect with the Docker client.. I'll remove this change.